2023-07-04 07:41:46.057 -03 [7056] LOG:  starting PostgreSQL 14.7, compiled by Visual C++ build 1914, 64-bit
2023-07-04 07:41:46.058 -03 [7056] LOG:  listening on IPv6 address "::", port 5432
2023-07-04 07:41:46.058 -03 [7056] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2023-07-04 07:41:46.188 -03 [7512] LOG:  database system was shut down at 2023-07-04 07:38:41 -03
2023-07-04 07:41:46.225 -03 [7056] LOG:  database system is ready to accept connections
2023-07-04 07:57:35.635 -03 [3204] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 07:57:35.635 -03 [3204] STATEMENT:  INSERT INTO Modulos (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 520, 'FUTURO', 12566, 22566, 15);
2023-07-04 07:58:07.257 -03 [3204] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 07:58:07.257 -03 [3204] STATEMENT:  INSERT INTO modulos (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 520, 'FUTURO', 12566, 22566, 15);
2023-07-04 07:58:15.667 -03 [3204] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 07:58:15.667 -03 [3204] STATEMENT:  INSERT INTO MODULOS (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 520, 'FUTURO', 12566, 22566, 15);
2023-07-04 07:59:09.995 -03 [3204] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 07:59:09.995 -03 [3204] STATEMENT:  INSERT INTO MODULOS (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 520, 'FUTURO', 12566, 22566, 15);
2023-07-04 07:59:35.184 -03 [3204] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 07:59:35.184 -03 [3204] STATEMENT:  INSERT INTO MODULOS (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 520, 'FUTURO', 1256, 2256, 15);
2023-07-04 07:59:52.796 -03 [12708] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 07:59:52.796 -03 [8564] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 07:59:52.796 -03 [3204] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:01:33.690 -03 [15296] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 08:01:33.690 -03 [15296] STATEMENT:  INSERT INTO MODULOS (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 520, 'FUTURO', 1256, 2256, 15);
2023-07-04 08:02:21.878 -03 [15296] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 08:02:21.878 -03 [15296] STATEMENT:  INSERT INTO MODULOS (ID, MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM, ATUALIZACAO) 
	VALUES (1, 1, 520, 'FUTURO', 1256, 2256, 15);
2023-07-04 08:04:36.008 -03 [16852] FATAL:  database "DBHIDROMETRO" does not exist
2023-07-04 08:05:35.331 -03 [8912] ERROR:  database "DBHidrometro" is being accessed by other users
2023-07-04 08:05:35.331 -03 [8912] DETAIL:  There is 1 other session using the database.
2023-07-04 08:05:35.331 -03 [8912] STATEMENT:  ALTER DATABASE "DBHidrometro" RENAME TO "DBHIDROMETRO";
2023-07-04 08:05:49.882 -03 [15296] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:05:49.882 -03 [11800] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:05:49.882 -03 [6980] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:05:49.882 -03 [3112] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:05:49.882 -03 [8912] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:10:05.586 -03 [16932] ERROR:  relation "modulos" does not exist at character 13
2023-07-04 08:10:05.586 -03 [16932] STATEMENT:  INSERT INTO MODULOS (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM) 
	VALUES (1, 520, 'FUTURO', 1256, 2256);
2023-07-04 08:12:58.415 -03 [16932] ERROR:  syntax error at or near "\" at character 13
2023-07-04 08:12:58.415 -03 [16932] STATEMENT:  INSERT INTO \"MODULOS\" (MODULO, PULSOS, LOCALIZACAO, INICIAL, FIM) 
	VALUES (1, 520, 'FUTURO', 1256, 2256);
2023-07-04 08:20:11.253 -03 [16932] ERROR:  syntax error at or near "SOUZA" at character 50
2023-07-04 08:20:11.253 -03 [16932] STATEMENT:  INSERT INTO TESTE (NOME, SOBRE) 
	VALUES (EDUARDO SOUZA);
2023-07-04 08:20:54.772 -03 [16932] ERROR:  relation "teste" does not exist at character 13
2023-07-04 08:20:54.772 -03 [16932] STATEMENT:  INSERT INTO TESTE (NOME, SOBRE) 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:22:54.009 -03 [6268] ERROR:  relation "teste" does not exist at character 13
2023-07-04 08:22:54.009 -03 [6268] STATEMENT:  INSERT INTO TESTE (NOME, SOBRE) 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:23:19.594 -03 [6268] ERROR:  column "nome" of relation "TESTE" does not exist at character 22
2023-07-04 08:23:19.594 -03 [6268] STATEMENT:  INSERT INTO "TESTE" (NOME, SOBRE) 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:24:07.909 -03 [6268] ERROR:  column "nome" of relation "TESTE" does not exist at character 22
2023-07-04 08:24:07.909 -03 [6268] STATEMENT:  INSERT INTO "TESTE" (nome, SOBRE) 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:24:30.357 -03 [6268] ERROR:  column "nome" of relation "TESTE" does not exist at character 29
2023-07-04 08:24:30.357 -03 [6268] STATEMENT:  INSERT INTO public."TESTE" (nome, SOBRE) 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:25:07.071 -03 [6268] ERROR:  syntax error at or near "t" at character 28
2023-07-04 08:25:07.071 -03 [6268] STATEMENT:  INSERT INTO public."TESTE" t (t."NOME", t."SOBRE") 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:25:28.725 -03 [6268] ERROR:  column "eduardo" does not exist at character 56
2023-07-04 08:25:28.725 -03 [6268] STATEMENT:  INSERT INTO public."TESTE"  ("NOME", "SOBRE") 
	VALUES (EDUARDO, SOUZA);
2023-07-04 08:25:47.769 -03 [6268] ERROR:  null value in column "ID" of relation "TESTE" violates not-null constraint
2023-07-04 08:25:47.769 -03 [6268] DETAIL:  Failing row contains (null, E, S).
2023-07-04 08:25:47.769 -03 [6268] STATEMENT:  INSERT INTO public."TESTE"  ("NOME", "SOBRE") 
	VALUES ('EDUARDO', 'SOUZA');
2023-07-04 08:34:31.044 -03 [16932] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.044 -03 [10012] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.044 -03 [9568] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.044 -03 [5160] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.045 -03 [14344] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.046 -03 [11620] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.046 -03 [16412] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.044 -03 [9128] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.046 -03 [14316] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:34:31.046 -03 [6268] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 08:38:14.102 -03 [16672] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:10:41.485 -03 [17788] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:10:41.485 -03 [12468] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:10:41.485 -03 [10136] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:10:41.485 -03 [4624] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:10:41.485 -03 [14784] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:12:27.886 -03 [6068] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:12:27.887 -03 [7236] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:20:25.042 -03 [4152] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:20:25.042 -03 [10844] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:20:25.043 -03 [12704] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:20:25.043 -03 [13852] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:20:25.044 -03 [2432] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 09:20:25.056 -03 [6276] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 11:55:46.511 -03 [18768] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 11:55:46.521 -03 [17440] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 11:55:46.524 -03 [17876] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 11:55:46.525 -03 [11056] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 11:55:46.525 -03 [2544] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 11:55:46.606 -03 [12008] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 16:54:24.710 -03 [7280] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 16:54:24.710 -03 [1124] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 16:54:24.711 -03 [17568] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 16:54:24.714 -03 [11304] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 20:57:01.494 -03 [6636] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 20:57:01.494 -03 [12752] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 20:57:01.494 -03 [13632] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 21:01:18.802 -03 [13652] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 21:01:18.803 -03 [8568] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2023-07-04 21:01:18.803 -03 [18852] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
